LICENSES

```diff
@@ -128,12 +128,17 @@ github.com/x448/float16,MIT
 github.com/xlab/treeprint,MIT
 go.opentelemetry.io/auto/sdk,Apache-2.0
 go.opentelemetry.io/otel,Apache-2.0
+go.opentelemetry.io/otel,BSD-3-Clause
 go.opentelemetry.io/otel/exporters/prometheus,Apache-2.0
 go.opentelemetry.io/otel/exporters/prometheus,BSD-3-Clause
 go.opentelemetry.io/otel/metric,Apache-2.0
+go.opentelemetry.io/otel/metric,BSD-3-Clause
 go.opentelemetry.io/otel/sdk,Apache-2.0
+go.opentelemetry.io/otel/sdk,BSD-3-Clause
 go.opentelemetry.io/otel/sdk/metric,Apache-2.0
+go.opentelemetry.io/otel/sdk/metric,BSD-3-Clause
 go.opentelemetry.io/otel/trace,Apache-2.0
+go.opentelemetry.io/otel/trace,BSD-3-Clause
```

Comment:
@inteon, any idea why we now have two licenses for OTEL deps here? Is this the expected behavior?